### PR TITLE
Pull request for Awesome books, dynamics check!

### PR DESCRIPTION
**Pull request by CrystallineButterfly**
Hi there, 

This is a project completed by me and @NatiDeme, 
So we have created an awesome books project where you can add and remove values: the values being the title and author of the book, this can be added in both the HTML displayed on the page and in local storage!

Thank you for the code review, I hope you are having a good day!

All the best-K

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,39 +20,27 @@
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
```
Comment (dasileker) [inline README.md:21]: - [ ] Please add directions on how to clone and run the app locally.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @NatiDeme,** 

### **Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!

## **Highlights**
- codes are clean ✅.
- Linters check ✅.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (CrystallineButterfly) [reply README.md:21]: Ok @dasileker Thank you, will do

**Comment by CrystallineButterfly** [comment]
Hi, there @ShoiraTa Thank you very much for the awesome review!

giphy (1)

Enjoy the rest of your day!

All the best-K

**Pull request merged**